packages/sdk-codegen/src/python.gen.spec.ts

```diff
@@ -1025,33 +1025,43 @@ class MergeFields(model.Model):
         const oneItem = [1]
         const threeItems = ['Abe', 'Zeb', token]
         const inputs: IDictionary<any> = {
+          /**
+           * The below key is quoted in the generated dictionary. This is a bug
+           * that might need addressing later on.
+           */
```

Comment:
This can wait for another PR, but can you explain what's going on here?

Reply:
In Python it is possible to have an integer as a dictionary key. Consider
```python
x = { 1: "one", "2": "two" }
```
These two values would be accessed with `x[1]` and `x["2"]` respectively in Python. However, in the Python code generator we are quoting all dictionary keys.